Content.Server/StationEvents/Components/VentClogRuleComponent.cs

```diff
@@ -1,14 +1,60 @@
 ﻿using Content.Server.StationEvents.Events;
+using Robust.Shared.Audio;
 
 namespace Content.Server.StationEvents.Components;
 
 [RegisterComponent, Access(typeof(VentClogRule))]
 public sealed class VentClogRuleComponent : Component
 {
+    /// <summary>
+    /// Somewhat safe chemicals to put in foam that probably won't instantly kill you.
+    /// There is a small chance of using any reagent, ignoring this.
+    /// </summary>
     [DataField("safeishVentChemicals")]
     public readonly IReadOnlyList<string> SafeishVentChemicals = new[]
     {
         "Water", "Blood", "Slime", "SpaceDrugs", "SpaceCleaner", "Nutriment", "Sugar", "SpaceLube", "Ephedrine", "Ale", "Beer"
     };
 
+    /// <summary>
+    /// Sound played when foam is being created.
+    /// </summary>
+    [DataField("sound")]
+    public SoundSpecifier Sound = new SoundPathSpecifier("/Audio/Effects/extinguish.ogg");
+
+    /// <summary>
+    /// The standard reagent quantity to put in the foam, modfied by event severity.
+    /// </summary>
+    [DataField("reagentQuantity"), ViewVariables(VVAccess.ReadWrite)]
+    public int ReagentQuantity = 200;
+
+    /// <summary>
+    /// The standard spreading of the foam, not modfied by event severity.
+    /// </summary>
+    [DataField("spread"), ViewVariables(VVAccess.ReadWrite)]
+    public int Spread = 20;
+
+    /// <summary>
+    /// How long the foam lasts for
+    /// </summary>
+    [DataField("time"), ViewVariables(VVAccess.ReadWrite)]
+    public float Time = 20f;
+
+    /// <summary>
+    /// Reagents that gets the weak numbers used instead of regular ones.
+    /// </summary>
+    [DataField("weakReagents"), ViewVariables(VVAccess.ReadWrite)]
```

Comment:
shouldnt this use a prototype list type serializer (make safeishVentChemicals use it too since that got past review apparently)

Reply:
o yeah good catch